### PR TITLE
mariadb: packaging, config and misc improvements

### PR DESCRIPTION
**Pull request by micmac1**
Maintainer: me
Compile tested: ar71xx, dlink dir-825-c1, master
Run tested: ar71xx, dlink dir-825-c1, master

Description:
Hello all,

This pull request improves packaging and configuration of mariadb.

Kind regards,
Seb

### CODE REVIEW

File: utils/mariadb/Makefile [hunk 1]
```diff
@@ -9,10 +9,11 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=mariadb
 PKG_VERSION:=10.2.19
-PKG_RELEASE:=2
+PKG_RELEASE:=3
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL := \
+	https://sources.openwrt.org \
```
Comment (hnyman) [inline utils/mariadb/Makefile]: I don't think that you should add  sources.openwrt.org to the download site list.

It serves as a fallback option in any case, but should not be listed separately for each package.

Comment (micmac1) [reply utils/mariadb/Makefile]: Hello Hannu,

OK :) I removed it.

Kind regards,
Seb

**Comment by micmac1** [comment]
Whoops, that introduced some patching artifact. Let me fix that.

**Comment by micmac1** [comment]
OK, fixed it.

**Pull request merged**